src/file-processor.ts

```diff
@@ -40,14 +41,26 @@ export async function* processDeploymentsInFile(
  * @public
  */
 export async function* processDeploymentsInStream(
-  stream: NodeJS.ReadableStream
-): AsyncIterable<DeploymentWithAuthChain> {
+  stream: NodeJS.ReadableStream,
+  logger: ILoggerComponent.ILogger
+): AsyncIterable<SyncDeployment> {
   for await (const line of processLineByLine(stream)) {
     const theLine = line.trim()
     if (theLine.startsWith('{') && theLine.endsWith('}')) {
-      const deployment = coerceEntityDeployment(JSON.parse(theLine))
-      if (deployment) {
-        yield deployment
+      const parsedLine = JSON.parse(theLine)
+      if (SnapshotSyncDeployment.validate(parsedLine)) {
+        yield parsedLine
+      } else if (PointerChangesSyncDeployment.validate(parsedLine)) {
+        yield parsedLine
+      } else {
+        const errors = {
+          ...SnapshotSyncDeployment.validate.errors,
```

Comment:
are those global vars or something like that?

Reply:
No, that's where the errors come from in the types exported by `common-schemas`.